ovirt-engine.spec.in

```diff
@@ -329,6 +329,10 @@ Requires:	ovirt-cockpit-sso
 Obsoletes:	ovirt-engine-api-explorer
 %endif
 
+# fapolicyd should be installed but does not have to be running
+# needed for OpenScap DISA STIG compliance
+Requires:	fapolicyd >= 1.1
```

Comment:
As we don't know when fapolicyd-1.1 will be released on EL9, please use conditional require:

```
# fapolicyd should be installed but does not have to be running
# needed for OpenScap DISA STIG compliance
# 
# TODO: remove condition when fapolicyd-1.1 is available on CS9
%if 0%{?rhel} < 9
Requires:	fapolicyd >= 1.1
%else
Requires:	fapolicyd
%endif
```

As we don't need DISA STIG functionality on upstream right now, it should be OK

Reply:
done